python-frontend/src/test/java/org/sonar/python/tree/PythonTreeMakerTest.java

```diff
@@ -1215,6 +1215,10 @@ public void try_statement() {
     assertThat(tryStatement.elseClause().body().statements()).hasSize(1);
     assertThat(tryStatement.children()).hasSize(5);
 
+    astNode = p.parse("try: pass\nexcept (RuntimeError, TypeError, NameError): pass\nelse: pass");
+    tryStatement = treeMaker.tryStatement(astNode);
+    assertThat(tryStatement.tryKeyword().value()).isEqualTo("try");
```

Comment:
What's the point of that test? Should we add more assertions to it?

Reply:
This shouldn't have been committed, my mistake!